python/cudf/cudf_pandas_tests/test_cudf_pandas.py

```diff
@@ -1795,3 +1795,21 @@ def test_iter_doesnot_raise(monkeypatch):
         monkeycontext.setenv("CUDF_PANDAS_FAIL_ON_FALLBACK", "True")
         for _ in s:
             pass
+
+
+@pytest.mark.timeout(5)
```

Comment:
Can we test this with `time.time()` instead of adding a dependency?

Is there a way to test this that doesn't require timing at all? Perhaps by mocking/augmenting the transform function to see how many times it is called?

Reply:
Done.